test/functional/feature_asset_locks.py

```diff
@@ -503,11 +502,13 @@ def test_withdrawal_limits(self, node_wallet, node, pubkey):
         node.generate(1)
         self.sync_all()
         credit_pool_balance_1 = self.get_credit_pool_balance()
-        assert_greater_than(credit_pool_balance_1, 10_900 * COIN)
+        assert_greater_than(credit_pool_balance_1, 10_100 * COIN)
```

Comment:
> because withdrawals are going to be above 10% limit after the first mined block

yep, that's true, thanks for suggestion: the test becomes more strict with this changes.